docs/interpreter/pig.md

```diff
@@ -30,6 +30,8 @@ group: manual
   - MapReduce
   - Tez_Local (Only Tez 0.7 is supported)
   - Tez  (Only Tez 0.7 is supported)
+  - Spark_Local (Only Spark 1.6.x is supported, by default it is Spark 1.6.3)
+  - Spark (Only Spark 1.6.x is supported, by default it is Spark 1.6.3)
 
 ## How to use
 
```

Comment:
ok, to clarify, in `How to setup Pig` it has words like `Spark Local Mode` and `Spark Mode` - it doesn't say to use `Spark Local Mode` you need to set `zeppelin.pig.execType` to `spark_local` with the `_` 

and in `Supported runtime mode` it doesn't point to `zeppelin.pig.execType`

i think it'll help to organizing a few of these together